src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let money = days * 40;
```

Comment:
Replace the literal '40' with a named constant to explain what this value represents. For example, 'const DAILY_RATE = 40;'.